include/revng/Pipes/PipelineManager.h

```diff
@@ -149,10 +150,12 @@ class PipelineManager {
   runAnalysis(llvm::StringRef AnalysisName,
               llvm::StringRef StepName,
               const pipeline::ContainerToTargetsMap &Targets,
+              const llvm::StringMap<std::string> &ExtraArgs = {},
               llvm::raw_ostream *DiagnosticLog = nullptr);
 
   /// Run all analysis in reverse post order (that is: parents first),
-  llvm::Expected<pipeline::DiffMap> runAllAnalyses();
+  llvm::Expected<pipeline::DiffMap>
+  runAllAnalyses(const llvm::StringMap<std::string> &ExtraArgs = {});
```

Comment:
I'm confused by `Extra`. Also, elsewhere you call them options, not args.
I'd drop extra and pick one among arguments and options.

Reply:
Not fixed.

Reply:
ahh forgot to change argument names outside of invokables.h

Reply:
Can you fix these? I'd like to merge this PR.